wallet/i18n/de.json

```diff
@@ -247,6 +250,8 @@
   "save": "Speichern",
   "submit": "Einreichen",
   "place_your_recovery_phrase_in_correct_order": "Platzieren Sie Ihre Genesung Phrase in der richtigen Reihenfolge.",
+  "update_now": "Update nu",
```

Comment:
This translation is wrong.

Reply:
@heyitsjk can you confirm this because this is what it is in the files that you provided.